src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -243,20 +244,68 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     return null;
   }
 
-  renderMatchNoUrl() {
+  renderHovercard() {
+    const error = this.match.error;
+    const url = this.match.attemptedUrl;
+    const {frame} = this.props;
     const {config} = this.match;
+    return (
+      <React.Fragment>
+        <StyledHovercard
+          header={
+            error === 'stack_root_mismatch' ? (
+              <span>{t('Mistmatch between filename and stack root')}</span>
```

Comment:
@MeredithAnya It would be great if we could link to docs here since the feedback we've gotten is that "stack root" is confusing. The examples we have in docs I think help explain what it means.

Reply:
@scefali Do you think it would be fine to have the docs link in the main message? so like 

"Configure Stack Trace Link to fix the problem or Visit Our Docs" (i dk something like that)

Reply:
@MeredithAnya That's fine but the ideal would be able to link to this specific error in a troubleshooting section (ex for AWS Lambda: https://docs.sentry.io/product/integrations/aws-lambda/#error-sentryrole-already-exists). But we don't have that section for Github so I think any link to the docs would be OK.

Reply:
we don't have a troubleshooting section yet, so i imagine it would be pretty easy to sift through once we create that section, since it shouldn't be too long.